.github/workflows/rebase.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           fetch-depth: 0 # otherwise, you will fail to push refs to dest repo
       - name: Automatic Rebase
-        uses: cirrus-actions/rebase@1.7
+        uses: cirrus-actions/rebase@1.8
```

Comment:
This action is archived. I think we may not need this workflow anymore since GitHub now offers merging the target branch into the PR branch from the UI. I can remove this workflow in the next PR if we think we don't need it anymore.